tests/e2e/tests/regression/cli/cli.e2e.ts

```diff
@@ -91,3 +101,81 @@ test
         //Verify result
         await t.expect(cliPage.cliOutputResponseSuccess.count).eql(repeats, `CLI contains ${repeats} results`);
 });
+test
+    .meta({ rte: rte.standalone })
+    .before(async () => {
+        await acceptLicenseTerms();
+    })
+    ('Verify that working with logical DBs, user can not see 0 DB index in CLI', async t => {
+        await addRedisDatabasePage.addRedisDataBase(ossStandaloneConfig);
+        //Enter logical index
+        await t.click(addRedisDatabasePage.databaseIndexCheckbox);
+        await t.typeText(addRedisDatabasePage.databaseIndexInput, index, { paste: true });
+        //Click for saving
+        await t.click(addRedisDatabasePage.addRedisDatabaseButton);
+        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+        //Open CLI
+        await t.click(cliPage.cliExpandButton);
+        //Verify that user can not see 0 DB index in CLI
+        for ( const text of cliMessage ) {
+            await t.expect(cliPage.cliArea.textContent).contains(text, 'No DB index is displayed in the CLI message');
+        }
+        await t.expect(cliPage.cliDbIndex.visible).eql(false, 'No DB index before the > character in CLI is displayed');
+        await t.expect(cliPage.cliEndpoint.textContent).eql(databaseEndpoint, 'Database index 0 is not displayed in the CLI endpoint');
+});
+test
+    .meta({ rte: rte.standalone })
+    .before(async () => {
+        await acceptLicenseTerms();
+    })
+    .after(async () => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName + ` [${index}]`);
```

Comment:
Could you adapt the beforeEach work with compatible all cases?

Reply:
removed tests with logical db in the different file